nucypher-core-python/nucypher_core/__init__.pyi

```diff
@@ -3,13 +3,44 @@ from typing import List, Dict, Iterable, Optional, Mapping, Tuple, Set
 from .umbral import SecretKey, PublicKey, Signer, Capsule, VerifiedKeyFrag, VerifiedCapsuleFrag
 
 
+class Conditions:
+
+    def __init__(self, conditions: str):
+        ...
+
+    @classmethod
+    def from_string(cls, conditions: str) -> Conditions:
```

Comment:
What is the difference between these two constructors?

Reply:
There's a difference in Rust (allocating vs non-allocating), and this structure is just mapped here. I guess we could safely remove it from Python and JS bindings.